src/main/resources/db/migration/V1_46__add_procurement_id.sql

```diff
@@ -0,0 +1,2 @@
+alter table dynamic_framework_contract
+    add column procurement_id varchar(30);
```

Comment:
Should this be non-nullable? What do we do with the data already in the db?

Reply:
To capture chats: yes, should be required and unique -- for existing data, we can generate random nonsensical ids -- it's only relevant for the development and research environments at the moment